requirements/_check_avail_extras.py

```diff
@@ -0,0 +1,16 @@
+# import gcsfs
+import hydra
+import jsonargparse
+import matplotlib
+import omegaconf
+import rich
+import torchtext
+
+print(matplotlib.__version__)
+print(torchtext.__version__)
+print(omegaconf.__version__)
+print(hydra.__version__)
+print(jsonargparse.__version__)
+# todo
+# print(gcsfs.__version__)
+print(rich.__name__)
```

Comment:
rich people and rich packages go by name and other common silly packages by version 😆